content/version/1/4/code-of-conduct.md

```diff
@@ -28,34 +28,35 @@ include:
 Examples of unacceptable behavior by participants include:
 
 * The use of sexualized language or imagery and unwelcome sexual attention or
-  advances
+  advances after being advised of its unwelcomeness
```

Comment:
This is bizarre. If something like this happened in a workplace, there wouldn't be any question, the person would be fired. Why should people working on open source projects be more accommodating to unwelcome sexual behavior than any other workplace?

Reply:
This change does nothing to make open source projects more accommodating to unwelcome sexual behavior.  Rather, it allows a calm discussion to take place before people start being accused of sexual harassment.  If someone makes a dumb joke and another person finds it unfunny, they should say so before invoking rules.

Reply:
This is simple arithmetic, what was once a one strike rule for sexual misconduct is now a two strikes rule, so it is not overstating the point to say it makes it 100% more accommodating to unwelcome sexual advances.

This does not govern "dumb jokes", this governs dick pics and other forms of sexual harassment. If you have a believe this rule is too broad and could conceivably govern "dumb jokes" that should be a different change.

Reply:
Bob and Alice, two project contributors, go out for drinks at a nearby bar to unwind after a long day at a conference.  Alice has a little too much to drink and starts flirting with Bob.  Bob, being a stereotypical tech nerd, misreads the situation and does something that could be considered sexual harassment.  The next morning, Alice used the CoC to get Bob removed from the project and potentially blacklisted from related projects through Twitter and/or mailing list drama, instead of simply telling Bob "hey, we had a misunderstanding last night, don't do that again".  The correct time to invoke a sexual harassment clause is when Bob doesn't listen and sends a bouquet of roses to Alice's house with a creepy note.

One unfortunate incident does not define a person.  Everyone makes mistakes and misreads situations sometimes.  However, there are some people out there who would take advantage of the situation to serve their own agenda.

Reply:
I fully support this amendment, the original phrasing is hostile toward people with ASD, and other neurologically diverse persons who may not have the high level of social skill that seems to be expected of them from the original.

Reply:
It is up to project maintainers to adjudicate disputes. Potential CoC violations are always to be considered in context, and there are alternative ways of dealing with conflicts and violations without banning someone outright from the project. If a project maintainer wants to boot someone from a project they can do so with or without a code of conduct. Participation in a community is not a gods-given right.